code/game/turfs/open/asteroid.dm

```diff
@@ -27,6 +27,7 @@
 	var/dug = FALSE
 	/// Percentage chance of receiving a bonus worm
 	var/worm_chance = 30
+	rust_resistance = RUST_RESISTANCE_ORGANIC
```

Comment:
Make sure all of these are *before* new vars are declared

IE, put it here:
```dm
/turf/myturf
    name = "My turf"
    rust_resistance = RUST_RESISTANCE_ORGANIC
    var/worm_chance = 50
```
not here:
```dm
/turf/myturf
    name = "My turf"
    var/worm_chance = 50
    rust_resistance = RUST_RESISTANCE_ORGANIC
```